subsys/bluetooth/audio/aics.c

```diff
@@ -24,6 +24,8 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_aics);
 
+#define NOTIFY_RETRY_DELAY_MS 100
```

Comment:
Still not quire sure about using this hardcoded value compared to the connection interval (although it is way easier). 

100ms kind of seems like a high value if e.g. the connection interval is 7.5ms 

Not sure if we need to modify it. Perhaps it should be a Kconfig option like `CONFIG_BT_ASCS_ASE_BUF_TIMEOUT`?

Reply:
Let me add Kconfig option with default value of 40ms which is in the middle of `BT_GAP_INIT_CONN_INT_MIN` and `BT_GAP_INIT_CONN_INT_MAX` ;)